src/main/resources/messages.properties

```diff
@@ -127,14 +127,14 @@ privacy.your-info.email.p3=If you share your contact information through Benefit
 privacy.your-info.cookies.p1=Cookies are small text files that websites place on the computers and mobile devices of people who visit those websites. Pixel tags (web beacons) are small blocks of code placed on websites and emails.
 privacy.your-info.cookies.li1=We use cookies and other technologies like pixel tags to remember your preferences, enhance your online experience, and gather data on how you use our Sites to improve how we promote our content, programs, and events.
 privacy.your-info.cookies.li2=Your use of our Sites indicates your consent to such use of Cookies.
-privacy.your-info.cookies.p2=Most browsers are initially set up to accept HTTP cookies. If you want to restrict or block the cookies set by our Site, or any other site, you can do so through your browser setting. The ?Help? function in your browser should explain how. Alternatively, you can visit <a target="_blank" rel="noreferrer noopener" href="https://www.aboutcookies.org/">www.aboutcookies.org</a>, which contains comprehensive information on how to do this on various browsers. You will find general information about cookies and details on deleting cookies from your machine.
+privacy.your-info.cookies.p2=Most browsers are initially set up to accept HTTP cookies. If you want to restrict or block the cookies set by our Site, or any other site, you can do so through your browser setting. The 'Help' function in your browser should explain how. Alternatively, you can visit <a href="www.aboutcookie.org">www.aboutcookie.org</a>, which contains comprehensive information on how to do this on various browsers. You will find general information about cookies and details on deleting cookies from your machine.
```

Comment:
Is there a reason we don't want this link to open in a new tab? Without the target="_blank" it will open in the same window potentially causing the client to lose their place in their application?